buildkite/provider.go

```diff
@@ -139,6 +139,7 @@ func (tf *terraformProvider) Resources(context.Context) []func() resource.Resour
 		newTeamResource,
 		newTestSuiteResource,
 		newPipelineTeamResource,
+		NewPipelineTemplateResource,
```

Comment:
Should be lower case; it is internal to the package.

Reply:
Tweaked - case of existing resources too, which I think we can consolidateA